### PR TITLE
Fix validation for empty state

### PR DESCRIPTION
**Pull request by shadcn**
I found two issues while testing the `monetization` branch.

1. State is being validated even when empty.
2. `ServiceNotFoundException` was not handled. It was resulting in a fatal error.

This PR fixes both.

### CODE REVIEW

File: src/Installer/Form/ApigeeMonetizationConfigurationForm.php [hunk 1]
```diff
@@ -404,24 +407,42 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
...
    *
    * @param \Drupal\Core\Form\FormStateInterface $form_state
    *   The form state.
+   * @param String $property_name
+   *   The property name.
    *
-   * @return string
-   *   The state code. eg. CA.
+   * @return mixed
+   *   The value of the giving property.
```
Comment (jacine) [inline src/Installer/Form/ApigeeMonetizationConfigurationForm.php]: "given" instead of "giving" maybe?

**Review by jacine — CHANGES_REQUESTED** [review]
This is a great catch @arshad!  Thanks for finding it and fixing it! 🎉 👍 

I found a tiny typo in here, but otherwise this is looking good.

File: src/Installer/Form/ApigeeMonetizationConfigurationForm.php (hunk 1 above)
Comment (shadcn) [reply src/Installer/Form/ApigeeMonetizationConfigurationForm.php]: Fixed. Thanks @jacine 👍

**Review by jacine — APPROVED** [review]
Thanks @arshad!! 👏

**Pull request merged**